webpackConfig/build.js

```diff
@@ -0,0 +1,79 @@
+const webpack = require("webpack");
```

Comment:
The test config is totally separate, and we have a real `/webpack.config.js` that runs both `test` and `build` - see https://github.com/palantir/plottable/pull/3318/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR5 for why